### PR TITLE
[test] add timeout for calling ot-ctl

### PR DESCRIPTION
**Pull request by bukepo**
Add timeout for debugging stuck issue.

### CODE REVIEW

**Comment by bukepo** [comment]
The problem: https://github.com/openthread/openthread/pull/6362/checks?check_run_id=2207886182

**Review by wgtdkp — APPROVED** [review]
LGTM 👍

**Comment by jwhui** [comment]
The "hang issue" was fixed by https://github.com/openthread/openthread/pull/6378. Do we still want to merge this PR?

**Comment by bukepo** [comment]
I'd like to merge it, which would help debug future issues.

File: tests/integration/common.sh [hunk 1]
```diff
@@ -207,22 +207,26 @@ expect {
 EOF
 }
 
+call_ot_ctl() {
```
Comment (simonlingoogle) [inline tests/integration/common.sh:210]:
```suggestion
ot_ctl() {
```
nit

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

**Review by wgtdkp — APPROVED** [review]
LGTM :+1:

File: tests/integration/common.sh [hunk 2]
```diff
@@ -86,7 +86,7 @@ start_daemon() {
     fi
 
     sudo rm -rf ${OT_DAEMON_SETTINGS_PATH}
-    sudo "${OT_DAEMON}" "spinel+hdlc+uart://${NON_CCM_RCP}?forkpty-arg=1" > "${OT_DAEMON_LOG}" 2>&1 &
+    sudo "${OT_DAEMON}" -d7 "spinel+hdlc+uart://${NON_CCM_RCP}?forkpty-arg=1" > "${OT_DAEMON_LOG}" 2>&1 &
```
Comment (wgtdkp) [inline tests/integration/common.sh:89]:
```suggestion
    sudo "${OT_DAEMON}" -d7 -v "spinel+hdlc+uart://${NON_CCM_RCP}?forkpty-arg=1" > "${OT_DAEMON_LOG}" 2>&1 &
```

**Review by jwhui — APPROVED** [review]

**Pull request merged**